### DISCUSSION TITLE
Multiple modals

### DISCUSSION THREAD

**Discussion by blytung** [software-mansion/react-native-screens#2518] [General]
I don't know if this is a bug or expected behavior. 

But since I updated to v4 and have multiple modals in a stack and want to close one of them, it will close the next one under as well. It worked fine before as mentioned. 

When I'm swiping down the modal, it works and it just close one, but if I close it with navigation.goback() it close both. Goback function runs on js and the swipe is native I guess.

**Comment by kkafar** [comment]
Hey @blytung this is a bug & it has been already reported here: https://github.com/software-mansion/react-native-screens/issues/2525. You can track the progress on the issue there!
